dataprocess/dataprocess_test.go

```diff
@@ -0,0 +1,40 @@
+package dataprocess
+
+import (
+	"os"
+	"testing"
+
+	"github.com/edaniels/golog"
+	pc "go.viam.com/rdk/pointcloud"
+	"go.viam.com/test"
+)
+
+func TestWriteBytesToFile(t *testing.T) {
+	_ = golog.NewTestLogger(t)
+	tempDir, err := os.MkdirTemp("", "*")
+	defer os.RemoveAll(tempDir)
+	test.That(t, err, test.ShouldBeNil)
+	// Save a set of bytes to a file
+	actualBytes := []byte{1, 5, 8}
+	fileDest := tempDir + "test_bytes"
+	err = WriteBytesToFile(actualBytes, fileDest)
+	test.That(t, err, test.ShouldBeNil)
+	// Test that the file was actually written
+	readBytes, err := os.ReadFile(fileDest)
+	test.That(t, err, test.ShouldBeNil)
+	test.That(t, readBytes, test.ShouldResemble, actualBytes)
+}
+
+func TestWritePCDToFile(t *testing.T) {
+	_ = golog.NewTestLogger(t)
```

Comment:
I recommend moving this into a t.Run so that people don't need to move it into a t.Run when a second test case is added.